src/nm-checkpoint.c

```diff
@@ -607,6 +680,7 @@ nm_checkpoint_new (NMManager *manager, GPtrArray *devices, guint32 rollback_time
 	priv->rollback_timeout_s = rollback_timeout_s;
 	priv->created_at_ms = nm_utils_get_monotonic_timestamp_ms ();
 	priv->flags = flags;
+	priv->removed_devices = NULL;
```

Comment:
commonly we rely on the field to be pre-initialized to `%NULL`. I don't think this line is necessary.

Reply:
removed